Dockerfile

```diff
@@ -35,9 +36,8 @@ ARG TARGETOS TARGETARCH
 # install simapp, remove packages
 RUN GOOS=$TARGETOS GOARCH=$TARGETARCH make build
 
-
-# Final image, without build artifacts. `/base` already contains openssl, glibc and all required libs to start an app
-FROM gcr.io/distroless/base
```

Comment:
wasnt this added on purpose? why are we reverting?

Reply:
It was added in https://github.com/cosmos/cosmos-sdk/pull/14793 next to other improvements (which made the image lighter).

Turns out, users need to have the extra commands in the alpine image https://github.com/cosmos/cosmos-sdk/issues/15672.

Reply:
Thank you, nice change. `alpine:3` should be a great trade-off. Slim enough and you can install the rest via package management.